tensorflow/lite/tools/verifier.cc

```diff
@@ -106,9 +106,9 @@ bool VerifyStringTensorBuffer(const Tensor& tensor, const Buffer& buffer,
     return false;
   }
   offset += sizeof(int32_t);
-  for (int i = 1; i <= num_strings; i++, offset += sizeof(int32_t)) {
+  for (int i = 1; i <= static_cast<int>(num_strings); i++, offset += sizeof(int32_t)) {
```

Comment:
This is inconsistent with the other rewrites of the loop header throughout these PRs